### PR TITLE
r-adgoftest   r-backports  r-bdsmatrix  r-bit r-acepack  r-assertthat  r-base64enc  r-bh  r-bitops

### PR DESCRIPTION
**Pull request by johanneskoester**
This adds the R packages mentioned in the title.

### CODE REVIEW

**Comment by johanneskoester** [comment]
CircleCI seems to have problems with HTTP downloads. It fails to download the sources although I can do it here from my local machine (even in the train). I will try again later.

**Comment by johanneskoester** [comment]
@conda-forge/core your linter seems to be offline, and there is a backlog of recipes in the master branch. Is it correct to remove them in this PR, or should I rather keep them?

**Comment by johanneskoester** [comment]
@conda-forge/core can you please merge?

**Comment by johanneskoester** [comment]
@ocefpaf @jakirkham could you merge this and the other R-related PRs that pass the builds?

**Comment by ocefpaf** [comment]
@johanneskoester can you rebase against latest `master`? It is hard to follow what was feedstocked or not with the recipe removal you have here.

**Comment by johanneskoester** [comment]
Well, everything mentioned in the title is a new feedstock. Any removal is just such that I don't build those recipes that are in the master branch.

**Comment by johanneskoester** [comment]
I would do what you suggest, but it took me the whole day to get all these builds passing. CRAN is kind of slow at the moment, and this causes conda-build to timeout sometimes. So if it is possible, I would like to keep them as they are.

**Comment by ocefpaf** [comment]
I know they are passing you can skip the CIs.

**Comment by johanneskoester** [comment]
Damnit, I forgot the skipping. Can you merge now? I have no idea about this last deletion. Maybe that was introduced by the rebase?

**Comment by johanneskoester** [comment]
I'll start my weekend now (European time). I would really appreciate if the other PRs could be merged without rebasing. Thanks!

File: recipes/r-acepack/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+build:
+  skip: true  # [win32]
+  number: 0
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
```
Comment (ocefpaf) [inline recipes/r-acepack/meta.yaml]: Will these play nice with conda-smithy without the `# [not win32]`? I lost track of that.

Comment (johanneskoester) [inline recipes/r-acepack/meta.yaml]: Yes. I have fixed all remaining issues. There is also a test case now.

**Comment by jakirkham** [comment]
Sorry I missed this.

> ...there is a backlog of recipes in the master branch. Is it correct to remove them in this PR, or should I rather keep them?

For future reference, no it is not ok to remove the backlogged recipes. Doing this could result in those recipes never being converted to feedstocks.

Further there should be no reason to do this. The org's linter and org's CIs are already "trained" to ignore recipes that have already been included in `master`. So they won't be built or linted using org resources. More details in the FAQ also linked below.

xref: https://github.com/conda-forge/staged-recipes/pull/1373
xref: https://github.com/conda-forge/conda-forge-webservices/pull/53
ref: https://github.com/conda-forge/staged-recipes#11-why-are-there-recipes-already-in-the-recipes-directory-should-i-do-something-about-it

**Comment by johanneskoester** [comment]
I see, thanks for the clarification!
